### PR TITLE
Deploy cinder-volume service using LVM

### PR DESCRIPTION
**Pull request by simonpasquier**
The service is deployed on the compute node(s).

### CODE REVIEW

**Comment by simonpasquier** [comment]
This requires the following changes too:
- https://review.openstack.org/#/q/status:open+project:openstack/salt-formula-cinder+branch:master+topic:fix-enabled-tests
- https://github.com/tcpcloud/salt-formula-linux/pull/40

File: scripts/fuel_infra_install.sh [hunk 1]
```diff
@@ -12,5 +12,19 @@ salt '*' saltutil.sync_all
 
 sleep 5
 
+# Initialize the loopback device for LVM on all the Cinder volume nodes
+CINDER_LVM_FILE="/srv/cinder-volume"
+for CINDER_VOLUME_NODE in $(salt -C 'I@cinder:volume:enabled:True' --out txt test.true | sed 's/:.*$//'); do
+    if ! salt "$CINDER_VOLUME_NODE" service.status cinder-device-setup 2>/dev/null; then
```
Comment (cznewt) [inline scripts/fuel_infra_install.sh]: this should be handled by salt, can you add role device_setup in cinder service for this?

Comment (simonpasquier) [inline scripts/fuel_infra_install.sh]: good idea! I'll do it.

**Comment by simonpasquier** [comment]
This change will be made to stacklight-salt-model instead.

**Pull request closed without merging**